contracts/csv.sol

```diff
@@ -71,6 +73,15 @@ abstract contract CSVWriter {
         vm.ffi(inputs);
     }
 
+    function makeDir(string memory dirPath_) internal {
+        string[] memory inputs = new string[](3);
+        inputs[0] = makeDirScriptPath;
+        inputs[1] = "-f";
+        inputs[2] = dirPath_;
+
+        vm.ffi(inputs);
```

Comment:
Do we still need to use `ffi`? I remember @JGcarv mentioned new cheatcodes for writing to files were added to Foundry: https://book.getfoundry.sh/cheatcodes/